controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -286,59 +195,3 @@ TEST(SensorTests, Calibration) {
   EXPECT_FLOW_NEAR(readings.inflow, Sensors::PressureDeltaToFlow(kPa(1.1f)));
   EXPECT_FLOW_NEAR(readings.outflow, Sensors::PressureDeltaToFlow(kPa(0.01f)));
 }
-
-TEST(SensorTests, FlowDrift) {
```

Comment:
Looks like we don't have drift tests anymore, why?

Reply:
The new controller test checks it in a more "unit" way -- checking that the controller's volume computation matches FlowIntegrator.  I guess it's not in this PR but in #508 I add a proper check that FlowIntegrator's volume goes to 0 when there's a fixed flow error.  I couldn't add it earlier because my error detection was so bad.